scripts/Microsoft.DotNet.Cli.Build.Framework/project.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "NETStandard.Library": "1.0.0-rc2-23805",
+        "System.Diagnostics.Process": "4.1.0-rc2-23805"
```

Comment:
Was Process intentionally removed from the standard library? That seems like a pretty "standard library" kind of thing to me...

/cc @ericstj @davidfowl